application.js

```diff
@@ -7,12 +7,43 @@ var bodyParser = require('body-parser');
 var raincatcherUser = require('fh-wfm-user/lib/router/mbaas');
 var sessionInit = require('./lib/sessionInit');
 var adminRouter = require('./lib/routes/admin');
+var fhComponentMetrics = require('fh-component-metrics');
 
 // list the endpoints which you want to make securable here
 var securableEndpoints;
 
 var app = express();
 
+function metricsEnabled(metricsCfg) {
+  return metricsCfg.enabled === 'true' && metricsCfg.host && metricsCfg.port;
+}
+
+var metricsConf = {
+  host: process.env.METRICS_HOST,
+  port: process.env.METRICS_PORT,
+  title: 'auth-svc-' + process.env.FH_TITLE,
+  enabled: process.env.METRICS_ENABLED || 'false',
+  cpu_interval: process.env.METRICS_CPU_INTERVAL |0 || 2000,
+  mem_interval: process.env.METRICS_MEM_INTERVAL |0 || 1000
```

Comment:
@paolobueno Yeah, I've changed it now. I'm a little less concerned about other developers not knowing what it does, since it then gives them the opportunity to go find out! :)